romancal/ramp_fitting/ramp_fit_step.py

```diff
@@ -45,67 +39,36 @@ class RampFitStep (Step):
     reference_file_types = ['readnoise', 'gain']
 
     def process(self, input):
-
-        # Open input as a Roman DataModel (single integration; 3D arrays)
-        with Roman.datamodels.RampModel(input) as R_input_model:  # syntax ?
-
-            # Create a JWST DataModel (4D arrays) to populate
-            input_model = JWST.datamodels.RampModel()  # syntax ?
-
-            # Populate JWST model with correct sized arrays from Roman model
-            input_model.meta = R_input_model.meta
-            input_model.data = np.broadcast_to(R_input_model.data, (1,) +
-                                               R_input_model.data.shape)
-
-            input_model.pixeldq = R_input_model.pixeldq
-
-            input_model.groupdq = np.broadcast_to(R_input_model.groupdq, (1,) +
-                                                  R_input_model.groupdq.shape)
-
-            input_model.err = np.broadcast_to(R_input_model.err, (1,) +
-                                              R_input_model.err.shape)
-
-            input_model.refout = np.broadcast_to(R_input_model.refout, (1,) +
-                                                 R_input_model.refout.shape)
-
+        with rdd.open(input, mode='rw') as input_model:
+            input_model.data = input_model.data[np.newaxis, :]
+            input_model.data.dtype=np.float32
+            input_model.groupdq = input_model.groupdq[np.newaxis, :]
```

Comment:
Shouldn't the err array also be 'redimensionalized' here ? ( and refout, even though it is not currently used ?)